packages/accounts/src/msca/builder.ts

```diff
@@ -190,6 +196,17 @@ export class MSCABuilder {
         return result as unknown as DynamicMSCA<TProviderDecorators & PD> & AD;
       };
 
+      extendWithAccountLoupeMethods = <AD, PD>(
+        accountLoupeAddress: Address
+      ): DynamicMSCA<TProviderDecorators & PD> & AD & IAccountLoupe => {
+        return Object.assign(
+          this,
+          accountLoupeDecorators(this, accountLoupeAddress)
+        ) as unknown as DynamicMSCA<TProviderDecorators & PD> &
+          AD &
+          IAccountLoupe;
+      };
```

Comment:
with the change above, you shouldn't need this method anymore because you can just call `.extend` on the account itself at build. you can include that in the `createMSCA*` method or add it here to the builder's build method (ie. instead of `return new (class ...)`, do `const account = new (class ...); return account.extend(accountLoupeDecorator)`